transcendence-app/src/authorization/infrastructure/db/user-to-role.entity.ts

```diff
@@ -11,11 +12,19 @@ export interface UserToRoleData {
 }
 
 export class UserToRole {
-  id: string;
-  role: Role;
+  @IsString()
+  @IsUUID()
+  @IsNotEmpty()
+  id!: string;
+
+  @IsEnum(Role)
+  @IsNotEmpty()
+  role!: Role;
```

Comment:
suggestion: We could remove the `@IsNotEmpty` and `@IsString` decorators

question: What do you think about creating a `UserToRoleDto`?

Reply:
I don't like duplicated code, and UserToRole Dto would be the third object with that shape, and the second object that implements the usertoroledata interface. But I guess it's more readable if it exists, and so I have made the change you requested.